### PR TITLE
Bdog 1519

### PR DESCRIPTION
**Pull request by colin-lamed**

### CODE REVIEW

File: app/uk/gov/hmrc/cataloguefrontend/shuttering/ShutterOverviewController.scala [hunk 1]
```diff
@@ -61,9 +61,13 @@ class ShutterOverviewController @Inject() (
                                    }
                                    .map(ws => (env, ws))
                                }
-        hasGlobalPerm  = request.optUser.exists(_.groups.contains(catalogueConfig.shutterPlatformGroup))
-        killSwitchLink = if (hasGlobalPerm) Some(catalogueConfig.killSwitchLink(shutterType.asString, env.asString)) else None
-        page           = shutterOverviewPage(envAndCurrentStates.toMap, shutterType, env, killSwitchLink)
+        hasGlobalPerm  <-  Future.successful(false) // TODO update internal-auth-client - disabled banner for now
```
Comment (colin-lamed) [inline app/uk/gov/hmrc/cataloguefrontend/shuttering/ShutterOverviewController.scala:64]: Will address this in a following PR

File: app/uk/gov/hmrc/cataloguefrontend/shuttering/ShutterWizardController.scala [hunk 2]
```diff
@@ -185,7 +190,7 @@ class ShutterWizardController @Inject() (
                                                      .map(BadRequest(_))
                                                  )
                           }
-         // check has `shutter-platform` group or is authorized for selected services
+         // check has permission to shutter selected services (TODO only display locations that can be shuttered)
```
Comment (colin-lamed) [inline app/uk/gov/hmrc/cataloguefrontend/shuttering/ShutterWizardController.scala:193]: Will address this in a following PR

File: conf/application.conf [hunk 3]
```diff
@@ -18,31 +18,21 @@
 
 include "frontend.conf"
 
-play.http.session.cookieName = "catalogue"
```
Comment (colin-lamed) [inline conf/application.conf:21]: As discussed, this needs to be the same as internal-auth-frontend - it should be set globally for the enviroment (app-config-common/platops) - but the default (mdtp) should suffice

File: app/uk/gov/hmrc/cataloguefrontend/CatalogueController.scala [hunk 4]
```diff
@@ -63,8 +62,7 @@ class CatalogueController @Inject() (
   leakDetectionService         : LeakDetectionService,
   shutterService               : ShutterService,
   defaultBranchesService       : DefaultBranchesService,
-  verifySignInStatus           : VerifySignInStatus,
-  umpAuthActionBuilder         : UmpAuthActionBuilder,
+  auth                         : FrontendAuthComponents,
```
Comment (chotai) [inline app/uk/gov/hmrc/cataloguefrontend/CatalogueController.scala:65]: is this actually used?

**Review by chotai — APPROVED** [review]

**Pull request merged**